### ISSUE TITLE
Can we upgrade?

### ISSUE THREAD

**Issue by ferngarc** [import-io/s3-deploy#63]
https://github.com/import-io/s3-deploy/blob/17fb805cbec81dc6e9ec218f7eb58c0cf3d1a7b3/package-lock.json#L3165

Vulnerable: <1.10.2 - Patched: >=1.10.2 - Path: mocha@3.5.0 > growl@1.9.2

Can we update to version 1.10.2 or greater?

**Comment by RishikeshDarandale** [comment]
Similarly, Can we upgrade `aws-sdk` and `lodash`?

```
npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ s3-deploy [dev]                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ s3-deploy > aws-sdk > xmlbuilder > lodash                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ s3-deploy [dev]                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ s3-deploy > lodash                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 2 low severity vulnerabilities in 38576 scanned packages
  2 vulnerabilities require manual review. See the full report for details.
Exited with code 1
```

**Comment by jenweber** [comment]
I have a fork with npm security fixes that I'll test out and PR. I took a quick look at this and most issues are addressed with `npm audit fix` plus small changes to the codebase to accommodate for major upgrades.
